Homebrew/Archetypes/Fighter/Fighter (Runeguard).xml

```diff
@@ -0,0 +1,93 @@
+				<text>	The rune's power fades if it moves more than 100 feet away from you and can't be used by anyone other than you.</text>
+				<text>	Runic Augmentations. You learn three runic augmentations of your choice, which are detailed under "Runic Augmentations" below. Each augmentation is specific to its master rune; for instance, if you learn the Sokn augmentation of the Ild master rune, you don't gain the ability to use the Sokn augmentation with other master runes unless you also learn that specific combination.</text>
+				<text>	Once you activate one of your runic augmentations, you must finish a short or long rest before you can use it again.</text>
+				<text>	You learn two additional augmentations at 7th, 10th, 15th, and 18th level. Each time you learn new augmentations, you can choose to replace one you already know with a different one.</text>
+				<text>	Augmentation Activation. During your turn, you can activate a single runic augmentation by focusing on the rune's power. You can activate one of these augmentations on your turn using your bonus action to empower your attacks (Sokn), your reaction to entrench your defenses (Vard) or skillfully enhance another ability (Hagr) with your action.</text>
+				<text>	Activating a rune requires concentration, which lasts a number of rounds equal to your proficiency bonus + your Constitution modifier (minimum of 3). While you concentrate on a rune, you can cast spells but you can't concentrate on a rune and a spell at the same time. You gain the ability to concentrate on two runes at once at 10th level.</text>
+				<text>	Saving Throws. Some of your runic augmentations require your target to make a saving throw to resist the effects. The saving throw DC is calculated as follows:</text>
+				<text>	Runic save DC = 8 + proficiency bonus + your Constitution modifier</text>
+			</feature>
+			<feature optional="YES">
+				<name>Runeguard: Runic Augmentations</name>
+				<text>Below are the runic augmentations for each of the five master runes. In the ancient tongues, the augmentation can be said before or after the rune.</text>
+				<text/>
+				<text>Sokn</text>
+				<text>Attack Augmentation</text>
+				<text>You can activate a rune with the Sokn augmentation as a bonus action. A rune's power only affects the weapon it is inscribed on.</text>
+				<text>Ild. Once per turn, when you hit a creature with a weapon attack, you ignite the target. At the start of each of the burning creature's turns, it takes 1d4  re damage for each time you've burned it, and it can then make a Constitution saving throw, ending the e ect of all such burns on itself on a success. Alternatively, the burned creature, or a creature within 5 feet of it, can use an action to make a Wisdom (Medicine) check against your runic save DC, ending the e ect of all burns on it on a success.</text>
```

Comment:
```suggestion
				<text>Ild. Once per turn, when you hit a creature with a weapon attack, you ignite the target. At the start of each of the burning creature's turns, it takes 1d4 fire damage for each time you've burned it, and it can then make a Constitution saving throw, ending the effect of all such burns on itself on a success. Alternatively, the burned creature, or a creature within 5 feet of it, can use an action to make a Wisdom (Medicine) check against your runic save DC, ending the effect of all burns on it on a success.</text>
```